pages/[username]/edit-profile.tsx

```diff
@@ -44,10 +47,27 @@ export default function EditBioPage(props: { userData: User }) {
             <textarea
                 value={bio}
                 onChange={e => setBio(e.target.value)}
-                className="w-full border p-4 rounded-md text-xl"
+                className="w-full border p-4 rounded-md text-xl focus:outline-none focus:ring focus:border-blue-300 "
                 placeholder="Write a bio..."
             />
-            <div className="flex mt-2">
+
+            <div className="mt-8">
+                <div className="flex flex-row">
+                    <h2 className="up-ui-title mb-4">
+                        Private account?
```

Comment:
Change this text to be more accurate. It's not a private account, it's just not showing up in the feed

skip the explainer text by renaming it something like "hide posts from global feed"